indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUnusedAction.java

```diff
@@ -38,14 +39,19 @@
   @JsonIgnore
   private final Interval interval;
 
+  @JsonIgnore
```

Comment:
please add a Json serde test if not already present.

Reply:
added